docs/oid-info.md

```diff
@@ -58,11 +60,89 @@ the git ref that the workflow run was based upon.
 This specifies the username identity in the OtherName Subject Alternative Name, as
 defined by [RFC5280 4.2.1.6](https://datatracker.ietf.org/doc/html/rfc5280#section-4.2.1.6).
 
+
+### 1.3.6.1.4.1.57264.1.8 | Source URI
```

Comment:
In order to disambiguate between "Source URI" and "Source Repository URI", how about:

```suggestion
### 1.3.6.1.4.1.57264.1.8 | Source Repository URI
```